src/awsexplorer/regionNode.ts

```diff
@@ -17,22 +17,22 @@ import { AWSTreeNodeBase } from '../shared/treeview/nodes/awsTreeNodeBase'
  * an account's Lambda Functions and CloudFormation stacks for this region)
  */
 export class RegionNode extends AWSTreeNodeBase {
-    private info: RegionInfo
+    private region: Region
     private readonly childNodes: AWSTreeNodeBase[] = []
 
     public get regionCode(): string {
-        return this.info.regionCode
+        return this.region.id
     }
 
     public get regionName(): string {
-        return this.info.regionName
+        return this.region.description
```

Comment:
`description` is name? can description be renamed to name if that is what it actually is?

Reply:
I almost did this during the change. I'll do this in a separate (smaller, faster) PR and then merge it into this one.